src/pebble.py

```diff
@@ -121,36 +224,36 @@ def change_config(charm_state: CharmState, container: ops.model.Container) -> No
     """
     try:
         synapse.execute_migrate_config(container=container, charm_state=charm_state)
-        synapse.enable_metrics(container=container)
-        synapse.enable_forgotten_room_retention(container=container)
-        synapse.enable_serve_server_wellknown(container=container)
+        current_yaml = _get_synapse_config(container)
```

Comment:
I wonder if `current_yaml` is a good name. Maybe `current_synapse_config` ?

Reply:
It is better, renamed. Thanks!